.github/snapshot-mvn-settings.xml

```diff
@@ -0,0 +1,19 @@
+<settings>
+  <servers>
+    <server>
+<!--      <id>gitlab</id>-->
```

Comment:
Code comment something like (uncomment to use gitlab if artifactory doesn't work out)
Or delete